python_modules/dagster/dagster_tests/core_tests/storage_tests/utils/event_log_storage.py

```diff
@@ -1961,3 +1962,33 @@ def gen_op():
             )
 
             assert len(records) == 1
+
+    def test_last_asset_materialization_updates_on_observation(self, storage):
```

Comment:
maybe rename this test?

Seems like this is testing asset key presence, not materialization stuff.

Reply:
renamed ✅